examples/osgi-test-example-bnd/cnf/ext/central.maven

```diff
@@ -0,0 +1,45 @@
+org.osgi:org.osgi.test.junit5:0.10.0
```

Comment:
We should use macros for the version so we don't have to change `0.10.0` many times. Also use macros for the junit, assertj, bnd versions too.

See https://github.com/bndtools/bnd/blob/a00db24d6526bbacd2d81700de2c3c9cb8e5001a/cnf/ext/central.mvn#L70-L86 for example.